run-fridge-fill.py

```diff
@@ -2,7 +2,19 @@ def read_file(text_file):
   if ".txt" in text_file:
     recipe_list = open(text_file)
     opened_recipe_list = (recipe_list.read())
-    print opened_recipe_list
+    #print opened_recipe_list
```

Comment:
Do you want to this commented code here? Generally, commented code like this removed before committing to master.